staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -714,6 +715,11 @@ func (r *Request) Do() Result {
 
 	var result Result
 	err := r.request(func(req *http.Request, resp *http.Response) {
+		rest_log := fmt.Sprintf("Request: %s %s\n", req.Method, req.URL)
```

Comment:
I think we should use glog here rather than inventing a new logging library.

Also, this code is widely used by other clients.  We should disable verbose operation logging by default.

Reply:
@mml Thanks for your review, I completely agree with you.
This was a prototype and I just wanted to get feedback to be confident for this design and direction before spending much time and power.
Many folks give me positive feedback, so I will fix these points.
Thanks again anyways :-)

Reply:
I found round_tripper.go already outputs the same log, so it is fine to avoid adding this.